### DISCUSSION TITLE
FT4 mode

### DISCUSSION THREAD

**Discussion by g0fcu** [foldynl/QLog#302] [General]
Hi

I'm loving QLog and its modern UI. When I imported my log my FT4 QSO's were given a mode of MFSK, new FT4 QSO's are also given a mode of MFSK. I am using JTDX. Can FT4 be added to the list of modes and logged as FT4?

Could you also tell me where the sqlite database is stored and which table I should update to correct the mode on the imported QSO's.

Thanks & 73
Simon
G0FCU.

**Comment by foldynl** [comment]
Based on ADIF spec, FT4 is not mode but it is a submode of MFSK. More about this is described in #296 

the internal database is stored in the same place as the backups (see QLog [Wiki](https://github.com/foldynl/QLog/wiki/Autobackup)). The file name is qlog.db

**Please, do not update an internal database because it can break many other internal processes in QLog**

**Comment by g0fcu** [comment]
Thanks for the explanation. I did a search for FT4 in Discussions but not in Issues where you have responded with a clear explanation. I also note the mode and sub-mode fields which I hadn't noticed.
